frontend/src/components/Contexts/AlertContext.tsx

```diff
@@ -197,14 +250,23 @@ export const AlertProvider: FC<Props> = ({ children }) => {
                 <FailedMissionAlertContent missions={newFailedMissions} />,
                 AlertCategory.ERROR
             )
+            setListAlert(
+                AlertType.MissionFail,
+                <FailedMissionAlertListContent missions={newFailedMissions} />,
+                AlertCategory.ERROR
+            )
             setNewFailedMissions([])
         }
         // eslint-disable-next-line react-hooks/exhaustive-deps
     }, [newFailedMissions])
 
     useEffect(() => {
         const newBlockedRobotNames = enabledRobots
-            .filter((robot) => robot.status === RobotStatus.Blocked)
+            .filter(
+                (robot) =>
+                    robot.currentInstallation.installationCode.toLocaleLowerCase() ===
```

Comment:
Why is there added an installation filter here?

Reply:
Resolved when cleaning commit history